lib/floe/workflow/runner/docker.rb

```diff
@@ -39,6 +39,68 @@ def run!(resource, env = {}, secrets = {})
         ensure
           secrets_file&.close!
         end
+
+        def run_async!(resource, env = {}, secrets = {})
+          raise ArgumentError, "Invalid resource" unless resource&.start_with?("docker://")
+
+          image = resource.sub("docker://", "")
+
+          params = ["run", :detach, [:net, "host"]]
```

Comment:
As discussed, we should avoid `--net host`

 it will definitely require rewriting some of the examples, so I'm ok to merge for now, but then we need a followup issue to remove it.

Reply:
We should discuss this separately since this isn't new code.
Docker isn't used for production only development and localhost:3000 is 99% of the development use cases so we can remove this if we want but it will require significant changes for developers' workflows

Reply:
Extracted to https://github.com/ManageIQ/floe/issues/77